tests/src/test/java/org/eclipse/hono/tests/registry/CredentialsAmqpIT.java

```diff
@@ -59,6 +60,20 @@ public static void prepareDeviceRegistry(final VertxTestContext ctx) {
                                 IntegrationTestSupport.HONO_PWD)));
 
         client.connect().onComplete(ctx.completing());
+
+    }
+
+    /**
+     * Setup device registry.
+     *
+     * @param ctx The vert.x test context.
+     */
+    @BeforeEach
+    public void setupDeviceRegistry(final VertxTestContext ctx) {
+        helper.addTenantIdForRemoval(Constants.DEFAULT_TENANT);
+        helper.registry
+                .addTenant(Constants.DEFAULT_TENANT)
+                .onComplete(ctx.completing());
```

Comment:
can we use a tenant with a random ID instead?

Reply:
We could. However, that way I found some other issues. In any case, this is all over the place, and I think it makes sense to do this in another PR, to keep this one smaller.

Reply:
Ok, makes sense